core/modules/path/path.path.inc

```diff
@@ -18,6 +18,7 @@ function node_path_info() {
     'pattern description' => t('Default URL alias pattern (applies to all content types with blank URL alias patterns below)'),
     'pattern default' => 'content/[node:title]',
     'batch update callback' => 'node_path_bulk_update_batch_process',
+    'type delete callback' => 'node_path_type_delete_callback',
```

Comment:
Wondered about that. It wasn't batched in the pre-PR code either.
We batch the update callback because its a query, followed by a node load on each query result, followed by      path_save_automatic_entity_alias() on each node. Thats heavy. But this is just a single DB delete query, so maybe.